tests/test_hdfs.py

```diff
@@ -166,21 +182,26 @@ def setup_spark(hdfs_with_kerberos, setup_history_server, configure_security_spa
 def test_terasort_suite():
     jar_url = 'https://downloads.mesosphere.io/spark/examples/spark-terasort-1.1-jar-with-dependencies_2.11.jar'
 
+    data_dir = "hdfs:///users/alice"
+    terasort_in = "{}/{}".format(data_dir, "terasort_in")
+    terasort_out = "{}/{}".format(data_dir, "terasort_out")
+    terasort_validate = "{}/{}".format(data_dir, "terasort_validate")
+
     teragen_args=["--class", "com.github.ehiggs.spark.terasort.TeraGen"] + KERBEROS_ARGS
     utils.run_tests(app_url=jar_url,
-                    app_args="1g hdfs:///terasort_in",
+                    app_args="1g {}".format(terasort_in),
```

Comment:
If we implement a wrapper, we could also accept a `list` for the `app_args` and then pass `" ".join(app_args)` to `utils.run_tests()`.